docs/developer/extending/apps/quickstart/creating-app.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: Creating an app
+---
+
+## Summing up
+
+Before we continue, let's sum up what should we have up until this point:
+
+- a Saleor account assigned to an organization
+- a Saleor Dashboard for a project we want to extend with an app
+- Saleor CLI installed
+
+:::tip
+If at any point you feel you've got lost, feel free to peek into the corresponding [repository 🖥️](https://github.com/saleor/saleor-app-abandoned-checkouts).
+:::
+
+All good? Let's start building then 🚀!
+
+## Step 1. Spawning the App Template
+
+Saleor CLI provides developers with the possibility to use a ready-made example app. In this guide, we will utilize CLI to spawn such a template.
+
+In your terminal, `cd` to the path where you want to install the template and type in:
+
+```shell
+saleor app create abandoned-checkouts
+```
+
+:::note
+Under the hood, the `saleor app create` command uses the Saleor App Template repository. If you want to read up on it, go [here](developer/extending/apps/developing-apps/app-template.mdx).
+:::
+
+This command will install a Next.js app called `abandoned-checkouts` equipped with many tools that'll come in handy during the process of integration.
+
+After the successful installation, you have to `cd` into the `abandoned-checkouts` directory, and boot up the project using [pnpm](https://pnpm.io/). The command is:
+
+```shell
+pnpm dev
+```
+
+## Step 2. Updating the App Manifest
+
+[The App Manifest](developer/extending/apps/manifest.mdx) serves as the source of information about the Saleor app. It includes the app name, description, URL, as well as configuration for webhooks and extensions. The information is then consumed by other Saleor parties, such as the API and the Dashboard.
+
+:::note
+Don't worry, you don't need to fill the manifest yourself. The data is automatically drawn from the `package.json` file and the context in which the app is running.
+:::
```

Comment:
Probably not true anymore, its not using package.json, maybe just to get the version number. But maybe something like "in app-template generated by CLI, basic manifest is already generated"?